packages/flame_studio/lib/src/widgets/flame_studio.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flame/game.dart';
+import 'package:flame_studio/src/widgets/scaffold.dart';
+import 'package:flame_studio/src/widgets/settings_provider.dart';
+import 'package:flame_studio/src/widgets/toolbar/pause_button.dart';
+import 'package:flame_studio/src/widgets/toolbar/start_button.dart';
+import 'package:flutter/widgets.dart';
+
+class FlameStudio extends StatefulWidget {
+  const FlameStudio(this.child, {super.key});
+
+  final Widget child;
+
+  @override
+  State<StatefulWidget> createState() => _FlameStudioState();
+
+  static final List<Widget> toolbarLeft = [];
+  static final List<Widget> toolbarMiddle = [
```

Comment:
These widget are used on the Toolbar component, so defining them there directly would make more sense IMO.

If I would add a new icon to the toolbar, that would be the first place that I would look for.

Reply:
done